test/functional/tools/output_filter_select.xml

```diff
@@ -0,0 +1,66 @@
+<tool id="output_filter_select" name="output_filter_select" version="1.0.0">
+    <description>test for output filtering with selects</description>
+    <command><![CDATA[
+#if "A" in str($sel).split(",")
+    echo 'test' > A &&
+#end if
+#if "AA" in str($sel).split(",")
+    echo 'test' > AA &&
+#end if
+echo $sel 
```

Comment:
It's not clear to me what you're testing here. If you are not concerned about the string casting just put static values into the separate files ?